core/audits/uses-rel-preconnect.js

```diff
@@ -147,6 +149,13 @@ class UsesRelPreconnectAudit extends Audit {
       if (node.type === 'network' ) lcpGraphURLs.add(node.record.url);
     });
 
+    const fcpGraph =
+      await LanternFirstContentfulPaint.getPessimisticGraph(pageGraph, processedNavigation);
+    const fcpGraphURLs = new Set();
+    fcpGraph.traverse(node => {
+      if (node.type === 'network' ) fcpGraphURLs.add(node.record.url);
```

Comment:
Nit: I realized this is copied but can you fix in both places?
```suggestion
      if (node.type === 'network') fcpGraphURLs.add(node.record.url);
```

Reply:
https://eslint.org/docs/latest/rules/no-multi-spaces

Looks like the lint rule only flags if there are *multiple* spaces.